doc/api/path.md

```diff
@@ -279,6 +279,27 @@ path.format({
 // Returns: 'C:\\path\\dir\\file.txt'
 ```
 
+## `path.glob(path, pattern)`
```

Comment:
I am not sure about the name. it might be confusing with the existence of `fs.glob`

Reply:
What do you think of `matchesGlob`?

Reply:
What about `path.match` ?

Reply:
`matchesGlob` or `matchesPattern` SGTM

Reply:
I went with `matchGlob`, for parity with the RegExp-like functions (`<>.match`)

Reply:
Unresolving, as @MoLow is correct (@tniessen explained in his comment)